examples/nuclei_image_segmentation/Nuclei_Image_Segmentation.ipynb

```diff
@@ -0,0 +1,521 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Nuclei Image Segmentation Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we will implement a UNet to solve Kaggle's [2018 Data Science Bowl Competition](https://www.kaggle.com/c/data-science-bowl-2018). The challenge asks participants to find the location of nuclei from images of cells. In additional to `skorch`, this tutorial uses `matplotlib` for plotting, `torchvision` for image augmentation and Kaggle's offical cli to download data. Please follow [Kaggle's installation and configuration documentation](https://github.com/Kaggle/kaggle-api#installation) to install the kaggle cli. Next, navigate to the [competition page](https://www.kaggle.com/c/data-science-bowl-2018), click on \"Late Submission\" and accept the terms and conditions to get access to the data."
```

Comment:
This duplicates the instructions given in README, but are different (e.g. here `torchvision` is explicitly requested). I think there should be a unified introduction, same inside the notebook and in the README.

Also, I think the introduction should outline the approach that will be taken to solve the problem. In its current state, it only mentions that this is going to be a UNet model (by the way, a link to the UNet paper would be useful). Would be nice to explain in a nutshell what UNet is, which backbone network will be used for feature extraction, where the weights come from, how we will freeze them, etc.

Reply:
> by the way, a link to the UNet paper would be useful

:+1:

Then I don't believe that an explanation of UNet is still necessary.